host/sgx/create.c

```diff
@@ -816,6 +817,10 @@ oe_result_t oe_sgx_build_enclave(
     enclave->size = enclave_size;
     enclave->text = enclave_addr + oeimage.text_rva;
 
+    /* Save the FS/GS segment registers of host side */
+    enclave->host_fsbase = (uint64_t)oe_get_fs_register_base();
```

Comment:
Each thread will have different FS, GS values. It seems one thread's FS, GS value will overwrite another threads.

Reply:
Thanks for your catch, now fs/gs are saved to td.